index.fr.md

```diff
@@ -0,0 +1,71 @@
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include toc.html type="start" title="Table des matières" %}
+{:/}
+
+- This will be replaced by an automatically generated TOC when using Markdown formatting.
+{:toc}
+
+{::nomarkdown}
+{% include toc.html type="end" %}
+{:/}
+
+## Ressources d'enseignement {#train}
+
+[[Comment rendre vos présentations accessibles à tous]](/teach-advocate/accessible-presentations/)
+: Vous aide à rendre vos présentations, vos discussions, vos réunions, et vos formations accessibles à l'ensemble de votre public potentiel, y compris les personnes handicapées, entre autres.
+
+[[Programmes d'étude sur l'accessibilité Web : une structure pour créer vos propres cours]](/curricula/)
+: Fournit des programmes d'étude pour vous aider à créer des cours sur l'accessibilité Web pour différents rôles. Ils définissent les objectifs de l'apprentissage et fournissent des idées pour enseigner et évaluer les connaissances. Le programme "Introduction à l'accessibilité Web" est actuellement disponible. Des programmes supplémentaires seront disponibles en 2020 et en 2021.
+
+[[Créer des présentations et des formations sur l'accessibilité Web]](/teach-advocate/accessibility-training/)
+: Fournit des informations pour les orateurs, les conférenciers, les professeurs et autres présentateurs pour aider leur public à mieux comprendre l'accessibilité Web. Vous trouverez différentes informations allant des slides introductives pour les débutants dans l'accessibilité Web, aux statistiques et aux démos pour les formateurs expérimentés. (Note : la majorité de ce contenu date de 2013. La page [Programmes d'étude](/curricula/) renvoie à des ressources plus récentes.)
+
+[Démonstration : avant, après](https://www.w3.org/WAI/demos/bad/)
+: Montre un site Web inaccessible et une version améliorée de ce dernier. Inclut des annotations qui soulignent les principaux freins à l'accessibilité et les améliorations apportées, ainsi que les rapports d'évaluation pour le WCAG 2.0.
+{:.paragraph-like}
+
+## Ressources de promotion {#advocate}
+
+[[Contacter les organisations à propos des sites Web inaccessibles]](/teach-advocate/contact-inaccessible-websites/)
+: Encourage à signaler aux organisations que certains éléments sur leurs sites Web représentent un frein à l'accessibilité. Fournit des recommandations relatives à l'idendification de contacts clés, de la description du problème et au suivi éventuel. Inclut des conseils et des exemples d'e-mails.
+
+[[Les entreprises et l'accessibilité numérique]](/business-case/)
+: Examine les raisons pour lesquelles les entreprises devraient adopter l'accessibilité. Inclut les avantages tangibles et intangibles, et les risques si l'accessibilité n'est pas optimale. Explique de quelle manière l'accessibilité induit l'innovation, améliore votre marque, permet de conquérir plus de parts de marché, et minimise les risques juridiques.  
```

Comment:
- "induit l'innovation" > the original version is more cautious : "can drive innovation"
- "permet de conquérir plus de parts de marché" : "Market reach" is different from "market share". The article is more about "How addressing accessibility may increase the number of potential consumers who could buy your product" than performing better than your competitors on a particular market. I know the French version of the article says "Augmenter les parts de marché", but I think it is misleading and I would suggest an edit too.

My suggestion :
"Examine les raisons pour lesquelles les organisations devraient se préoccuper de l'accessibilité. Inclut des avantages tangibles et intangibles, et les risques si l'accessibilité n'est pas correctement prise en compte. Explique comment l'accessibilité peut être un moteur de l'innovation, améliorer votre marque, étendre son marché, et minimiser les risques juridiques."

What do you think ?